### PR TITLE
fix: add `--disable-dev-shm-usage`

### PR DESCRIPTION
**Pull request by svewag**
in order to bypass the limitation of 64 MB for the shared memory in
docker container.

https://docs.docker.com/engine/reference/run/ (see 'shm_size')

Since Chrome 75 for some reasons this size is not sufficient anymore
and causes weird errors when launching Chrome or running Lighthouse.
https://github.com/puppeteer/puppeteer/blob/master/docs/troubleshooting.md#tips

In our case suddenly the lightouse audits 'offline-start', 'service-worker' and
'works-offline' did fail.

### CODE REVIEW

File: lib/browser.js [hunk 1]
```diff
@@ -1,9 +1,24 @@
+import { log } from './logger'
```
Comment (cilice) [inline lib/browser.js:2]: we should probably decide if we go for require, as in Line 3 or with esm imports. Let's go esm?

**Review by cilice — COMMENTED** [review]
How comes `/dist` is committed inside the repository??

File: lib/browser.js (hunk 1 above)
Comment (svewag) [reply lib/browser.js:2]: sounds good

**Comment by svewag** [comment]
> How comes `/dist` is committed inside the repository??

historical reasons, didn't know it better at that time

**Comment by cilice** [comment]
@svewag Well, there is never a better time to fix this than right now, before doing a new release 💚

File: lib/browser.js (hunk 1 above)
Comment (svewag) [reply lib/browser.js:2]: @cilice, now it is esm

**Comment by svewag** [comment]
bundle is removed

**Review by cilice — APPROVED** [review]

**Pull request merged**